.github/workflows/ci.yml

```diff
@@ -31,10 +31,11 @@ jobs:
         run: |
           tools/ci/check_json.sh
           tools/ci/build_tgui.sh
-          python tools/ci/check_grep2.py
-          python tools/ci/check_line_endings.py
-          python tools/ci/check_file_names.py
-          python tools/ci/unticked_files.py ${GITHUB_WORKSPACE}
+          python3 tools/ci/check_grep2.py
```

Comment:
can you swap these back to `python`? there may be a `python3` symlink added by the setup-python action but i don't actually see that anywhere in its recipe and i don't trust it.

Reply:
Ah, you probably did that with the adding of the maplint. I didn't notice when deconflicting, I'll switch them all back to python, my bad